src/App.js

```diff
@@ -2,17 +2,44 @@ import React, { Component } from "react";
 
 import Bookings from "./Bookings";
 import Restaurant from "./Restaurant";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
 import "./App.css";
+const footerData = ['123 Fake Street, London, E1 4UD', 'hello@fakehotel.com', '0123 456789'];
+
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">CYF Hotel</header>
-        <Bookings />
-        <Restaurant />
-      </div>
-    );
+		<div className="App">
+			<Heading />
+			<div className="App-TouristInfoCards">
+				<TouristInfoCards
```

Comment:
This component name implies that it shows multiple cards but actually it shows just 1. That's not really a big problem but it would be nice to fix.

I would rename your existing component to `TouristInfoCard` (no "s" at the end) and then create a new component called `TouristInfoCards` which renders all 3 of the cards.